### PR TITLE
Merge 1.0.3 release

### PR DESCRIPTION
**Pull request by Areson**
Updating to prep for a new release to the forge.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,13 @@
+## 2019-??-?? - 1.0.3
+  - Added items to satisyf puppet-lin and pdk (Thanks alexjfisher)
```
Comment (alexjfisher) [inline CHANGELOG.md]: Was this really me?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,8 +1,13 @@
...
+  - Added better validation for verifying proper parameters passed (Thanks alexjfisher)
+  - Changed syntax on attributes used by module to satisfy REXML (Thanks MacGregor001)
```
Comment (alexjfisher) [inline CHANGELOG.md:4]: This changed isn't actually merged (or in this branch).

File: CHANGELOG.md (hunk 1 above)
Comment (Areson) [reply CHANGELOG.md]: Shoot, you are right. It was silug.

Comment (alexjfisher) [reply CHANGELOG.md]: and I think I've found another bug.  Just debugging it now.

File: CHANGELOG.md (hunk 2 above)
Comment (Areson) [reply CHANGELOG.md:4]: No, not yet. I was just staging these to get ready to wrap this up to into a release to push to the forge. The changes are elsewhere and I was using this branch as a placeholder.

File: CHANGELOG.md (hunk 1 above)
Comment (Areson) [reply CHANGELOG.md]: Quite possible. There was a slew of pull requests and at least one broke a few things. We got a correction back in for that, but it's possible other scenarios that the reporter had not reviewed were failing.

Comment (alexjfisher) [reply CHANGELOG.md]:
```
undefined method `key?' for nil:NilClass
/opt/puppetlabs/puppet/cache/lib/puppet/util/xml_file.rb:136:in `block in set_tag'
/opt/puppetlabs/puppet/lib/ruby/2.5.0/rexml/xpath.rb:68:in `each'
/opt/puppetlabs/puppet/lib/ruby/2.5.0/rexml/xpath.rb:68:in `each'
/opt/puppetlabs/puppet/cache/lib/puppet/util/xml_file.rb:112:in `set_tag'
/opt/puppetlabs/puppet/cache/lib/puppet/provider/xml_fragment/xml_fragment.rb:154:in `create'
```

I don't think this is necessarily a regression in the pdk PR though. In fact, I'm pretty sure it's not.  I'll open a PR with a fix.

Comment (alexjfisher) [reply CHANGELOG.md]: https://github.com/Areson/xml_fragment/pull/12

Comment (Areson) [reply CHANGELOG.md]: That one isn't, no. One of the previous PRs modified some class variable names and it wasn't propagated throughout the code properly and broke some lookups.

Comment (alexjfisher) [reply CHANGELOG.md]: yes, I saw that.  I've only hit these issues today as I'm trying to use `purge` for the first time.

**Comment by alexjfisher** [comment]
Any update on a release?

**Comment by Areson** [comment]
I'll try and get it out in the next few days. I'm super behind on a number of projects because at some point all of my GitHub notifications got routed to my spam folder and I didn't notice.

**Comment by Areson** [comment]
Merged changes from the development branch into the release branch and prepping for final merge.

**Pull request merged**